### PR TITLE
Fix simple typo: aliase -> alias

### PR DESCRIPTION
**Pull request by timgates42**

### CODE REVIEW

File: examples/aliases/aliases.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get_command(self, ctx, cmd_name):
         # will create the config object is missing.
         cfg = ctx.ensure_object(Config)
 
-        # Step three: lookup an explicit command aliase in the config
+        # Step three: lookup an explicit command alias in the config
```
Comment (jab) [inline examples/aliases/aliases.py]: Can you please change “lookup” to “look up” while you’re fixing this?

**Review by jab — CHANGES_REQUESTED** [review]

File: examples/aliases/aliases.py (hunk 1 above)
Comment (timgates42) [reply examples/aliases/aliases.py]: Sure thing

Comment (timgates42) [reply examples/aliases/aliases.py]: Thanks for spotting that - fixed

**Comment by jab** [comment]
Thank you for the contribution!

**Pull request merged**